Source/UETorch/Private/UETorch.cpp

```diff
@@ -706,3 +706,43 @@ extern "C" bool CaptureDepthField(UObject* _this, const IntSize* size, void* dat
 	}
 	return true;
 }
+
+extern "C" bool SetActorPosition(AActor* object, double x, double y, double z) {
+	return object->SetActorLocation(FVector(x,y,z), false);
+}
+
+extern "C" bool SetActorRotation(AActor* object, double pitch, double roll, double yaw) {
+	return object->SetActorRotation(FRotator(pitch,roll,yaw));
+}
+
+extern "C" void SetActorVisible(AActor* object, bool visible) {
+	object->SetActorHiddenInGame(!visible);
+}
+
+extern "C" bool SetActorVelocity(AActor* object, double x, double y, double z) {
+	if(!object) {
+		printf("object doesn't exist\n");
+		return false;
+	}
+	UStaticMeshComponent *component = object->FindComponentByClass<UStaticMeshComponent>();
+	if(!component) {
+		printf("Object doesn't have StaticMeshComponent\n");
+		return false;
+	}
+	component->SetPhysicsLinearVelocity(FVector(x,y,z));
+	return true;
+}
+
```

Comment:
Do you have an idea about how this should be implemented the doc for [SetPhysicsAngularVelocity](https://docs.unrealengine.com/latest/INT/API/Runtime/Engine/Components/UPrimitiveComponent/SetPhysicsAngularVelocity/index.html) suggests using [AddTorque](https://docs.unrealengine.com/latest/INT/API/Runtime/Engine/Components/UPrimitiveComponent/AddTorque/index.html) or [AddImpulse](https://docs.unrealengine.com/latest/INT/API/Runtime/Engine/Components/UPrimitiveComponent/AddImpulse/index.html), also all the three are receiving a BoneName variable.